StadiumSeating/MainWindow.xaml.cs

```diff
@@ -9,102 +9,89 @@ namespace StadiumSeating
     public partial class MainWindow : Window
     {
         int classATotal = 0, classBTotal = 0, classCTotal = 0;
+        
         int classATickets = 0, classBTickets = 0, classCTickets = 0;
+        
         public MainWindow()
         {
             InitializeComponent();
-            tbTotalTicketsClassA.Content = "$" + classATotal.ToString("n2");
-            tbTotalTicketsClassB.Content = "$" + classBTotal.ToString("n2");
-            tbTotalTicketsClassC.Content = "$" + classCTotal.ToString("n2");
-            tbTotalRevenueGenerated.Content = "$" + 0.ToString("n2");
+            
+            tbTotalTicketsClassA.Content = classATotal.ToString("C");
+            
+            tbTotalTicketsClassB.Content = classBTotal.ToString("C");
+            
+            tbTotalTicketsClassC.Content = classCTotal.ToString("C");
+            
+            tbTotalRevenueGenerated.Content = 0.ToString("C");
         }
 
+        static int EmptyStringChecker(string currentValue)
+        {
+            return currentValue == "" ? 0 : int.Parse(currentValue);
+        }
+
+        private void CalcTotalIncomeGenerated()
+        {
+            int totalTicketsSold = classATickets + classBTickets + classCTickets;
+
+            int totalIncomeGenerated = classATotal + classBTotal + classCTotal;
+
+            tbTotalTicketsSold.Content = totalTicketsSold.ToString();
+
+            tbTotalRevenueGenerated.Content = totalIncomeGenerated.ToString("C");
+        }
 
         private void CalcOnTextChange(object sender, RoutedEventArgs e)
         {
             try
             {
                 if (sender.Equals(txtTicketForClassA))
                 {
-                    if (txtTicketForClassA.Text.Length > 0)
-                    {
-                        classATotal = int.Parse(txtTicketForClassA.Text) * 15;
-
-                        classATickets = int.Parse(txtTicketForClassA.Text);
-
-                        tbTotalTicketsClassA.Content = "$" + classATotal.ToString("n2");
 
-                        errorMessageForClassA.Content = "";
+                    int userInput = EmptyStringChecker(txtTicketForClassA.Text);
```

Comment:
Creating a new variable for each means we only have to run the int.Parse() on the text once, then we can apply it to both the class*Total and class*Tickets.